### PR TITLE
Math: Remove closures part I.

### PR DESCRIPTION
**Pull request by Mugen87**
This is the first PR of a series of changes. My goal is to remove all IIFEs from the code base. This is a preparation for the actual long-term goal, the introduction of ES6 classes. Everything together will modernize the code base and improve tree-shaking capabilities in order to realize smaller bundle sizes. Both features were frequently requested by users.

I've started with just a few simple files. Helper variable declarations are now at the top of the modules. The naming convention of these variables are based on @bhouston's suggestion here https://github.com/mrdoob/three.js/pull/14695#issuecomment-414880386. A simple underscore as a prefix will make it much easier to detect module scope variables when reading the code (other/better suggestions are welcome^^). Besides, the variables are instantiated lazily when the methods are invoked.

### CODE REVIEW

File: src/math/Euler.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ import { _Math } from './Math.js';
  * @author bhouston / http://clara.io
  */
 
+var _matrix, _quaterion;
```
Comment (EliasHasle) [inline src/math/Euler.js]: This should be __quaternion_, right? :-)

Comment (Mugen87) [reply src/math/Euler.js]: You are so right!^^

**Comment by EliasHasle** [comment]
How do these module-scoped variables play with Webworkers? Will each worker have a separate instance?

**Comment by Mugen87** [comment]
The execution context of a web worker is isolated. So...well...I'm not sure what you are referring to^^

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**